pkg/controller/storagecluster/cephcluster.go

```diff
@@ -475,3 +484,29 @@ func newCephDaemonResources(custom map[string]corev1.ResourceRequirements) map[s
 
 	return resources
 }
+
+func (r *ReconcileStorageCluster) throttleStorageDevices(storageClassName string) (bool, error) {
+	storageClass := &storagev1.StorageClass{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: "", Name: storageClassName}, storageClass)
+	if err != nil {
+		return false, fmt.Errorf("failed to retrieve StorageClass %q. %+v", storageClassName, err)
+	}
+	bool flag := false
+	switch storageClass.Provisioner {
```

Comment:
This entire switch statement looks broken. Since you've removed the WIP tag, you should be running `make ocs-operator-ci` before ever pushing to GitHub to catch things like this.